.gitlab-ci.yml

```diff
@@ -202,7 +202,7 @@ test:backend:unit:
       artifacts: false
   rules:
     - changes:
-        paths: ["backend/**/*.go"]
+        paths: ["backend/**/*", "backend/go.mod"]
```

Comment:
Shouldn't we still filter by source files?
Now it will trigger unit tests even if you change acceptance test or other miscellaneous files too.
```suggestion
        paths: ["backend/**/*.go", "backend/go.mod"]
```

Reply:
yes, we should